moz_kinto_publisher/main.py

```diff
@@ -550,7 +555,7 @@ def publish_intermediates(*, args, rw_client, new_filter_run_id=None):
     remote_expired = set()
     for i in remote_only:
         try:
-            if remote_intermediates[i].is_expired():
+            if remote_intermediates[i].is_expired(rw_client=rw_client):
```

Comment:
Yeah I think there's a mismatch in this parameter name, assuming I understand Python.

Reply:
yeah the argument name is kinto_client. Thanks!